lib/6to5/transformation/modules/common.js

```diff
@@ -26,13 +27,11 @@ CommonJSFormatter.prototype.importSpecifier = function (specifier, node, nodes)
 
   // import foo from "foo";
   if (t.isSpecifierDefault(specifier)) {
-    nodes.push(t.variableDeclaration("var", [
```

Comment:
This has just broken module interop and wont work with modules that export a `default` property.